### ISSUE TITLE
How can I tell if a user sent several files at once?

### ISSUE THREAD

**Issue by Rafinol** [php-telegram-bot/core#763]
### Required Information

- Operating system:
- PHP version:
- PHP Telegram Bot version:
- Using MySQL database: yes / no
- MySQL version:
- Update Method: Webhook / getUpdates
- Self-signed certificate: yes / no
- RAW update (if available):

**Comment by jacklul** [comment]
Each file will be a seperated update as far as I know...

**Comment by noplanman** [comment]
Exactly. But the bot won't know if they were sent in 1 batch or multiple uploads after one another I think. Time will be the only way, and possible ID in sequence as well, in case the user types text commands in between.

@Rafinol What is your use-case exactly?

**Comment by Rafinol** [comment]
I want a person to send a message, and in the following message a description. If a person sends several pictures at once, it will not be clear to which image the description is being written

**Comment by noplanman** [comment]
@Rafinol Have you managed to find a smart solution for this?

**Comment by DonTisto** [comment]
Maybe the best way sends back files by one and wait description(message from user) after each of file?

**Comment by noplanman** [comment]
@DonTisto That's a nice solution :+1: 
So you're thinking the user can send many files, and then at the end make a description for each one?

**Comment by DonTisto** [comment]
@jacklul And then call second conversation for description as probably solution. I've used conversation exactly for this, but without description, just upload by tag.
@noplanman Yes, i think it's solving the problem with different upload speed of files.
But maybe better way use "conversation" + "conversation2"
First for get files, second for get descriptions. It gives users ability to put description later.
